src/status_im/ui/screens/wallet/collectibles/cryptokitties.cljs

```diff
@@ -46,9 +47,9 @@
 (defmethod collectibles/render-collectible ck [_ {:keys [id name bio image_url]}]
   [react/view {:style styles/details}
    [react/view {:style styles/details-text}
-    ;; TODO reenable image once SVG is supported
-    #_[react/image {:style {:width 80 :height 80 :margin 10 :background-color "red"} :source {:uri image_url}}]
-    [react/view {}
+    [svgimage/svgimage {:style styles/details-image
```

Comment:
So CK pngs will also use this? Does it display the same?
Or should we use images for those?

Reply:
yes, CK pngs will be displayed fine, svgimage supports pngs